app/models/user.rb

```diff
@@ -478,6 +478,10 @@ def receives_follower_email_notifications?
       email_follower_notifications
   end
 
+  def title
+    name
+  end
```

Comment:
Technically don't need to do this in this PR. I think the `title` method is only for Algolia, but not entirely sure.

I ran into an issue where creating a `vomit` reaction on a user in console caused a 500 in `/internal/reports`, so proactively fixing this now.

Reply:
makes sense!